### ISSUE TITLE
Investigate CI failures

### ISSUE THREAD

**Issue by stress-tess** [Bears-R-Us/arkouda#3060]
We're seeing CI failures. This is likely due to an issue with one of our dependencies:
https://github.com/Bears-R-Us/arkouda/actions/runs/8394087490/job/22995195053?pr=3057

**Comment by stress-tess** [comment]
Comparing with the last successful run: https://github.com/Bears-R-Us/arkouda/actions/runs/8382205567/job/22955499859

the `apache.jfrog` link that normally works is giving a `302 Moved Temporarily` html code. The redirected site tries to treat an html file as a `.deb`. This seems to be what's causing the error. I'm not sure how to go about fixing it but I think when that link becomes available again this error will probably resolve itself.

```
--2024-03-22 19:41:32--  https://apache.jfrog.io/artifactory/arrow/debian/apache-arrow-apt-source-latest-bullseye.deb
Resolving apache.jfrog.io (apache.jfrog.io)... 3.95.117.170, 18.214.194.113, 18.232.172.199
Connecting to apache.jfrog.io (apache.jfrog.io)|3.95.117.170|:443... connected.
HTTP request sent, awaiting response... 302 Moved Temporarily
Location: https://landing.jfrog.com/reactivate-server/apache [following]
--2024-03-22 19:41:32--  https://landing.jfrog.com/reactivate-server/apache
Resolving landing.jfrog.com (landing.jfrog.com)... 3.95.117.170, 18.214.194.113, 18.232.172.199
Connecting to landing.jfrog.com (landing.jfrog.com)|3.95.117.170|:443... connected.
HTTP request sent, awaiting response... 200 OK
Length: 11533 (11K) [text/html]
Saving to: 'apache-arrow-apt-source-latest-bullseye.deb'

     0K .......... .                                          100% 1.99M=0.006s

2024-03-22 19:41:32 (1.99 MB/s) - 'apache-arrow-apt-source-latest-bullseye.deb' saved [11533/11533]

E: Invalid archive signature
E: Internal error, could not locate member control.tar{.zst,.lz4,.gz,.xz,.bz2,.lzma,}
E: Could not read meta data from /__w/arkouda/arkouda/apache-arrow-apt-source-latest-bullseye.deb
E: The package lists or status file could not be parsed or opened.
Reading package lists...
Error: Process completed with exit code 100.
```

**Comment by stress-tess** [comment]
Yep! We're not the only ones encountering this. I found this issue in `apache/couch-db` but the same thing happens for `apache/arrow`

Navigating to https://landing.jfrog.com/reactivate-server/apache in a browser gives:
<img width="939" alt="Screenshot 2024-03-22 at 5 15 25 PM" src="https://github.com/Bears-R-Us/arkouda/assets/48131946/29a852c8-215d-48e6-b49b-a371f16d0b28">

It's worth noting the same issue happened last year to the date, so I'm guessing someone just forgot to renew a membership. It was fixed the next day, so I'm much more confident that this will be short lived
